compiler/rustc_borrowck/src/type_check/mod.rs

```diff
@@ -150,14 +150,14 @@ pub(crate) fn type_check<'a, 'tcx>(
         infcx,
         param_env,
         implicit_region_bound,
-        universal_regions,
```

Comment:
Rc clone()'s better be `Rc::clone`, so whenever field refactored out of Rc, compiler will warn, instead of wasting clone()'s.

Reply:
Submitted #131375, but is some discussion required before enabling it in CI?

Reply:
Seems likely